web/client/plugins/CRSSelector.jsx

```diff
@@ -0,0 +1,176 @@
+            filterAllowedCRS={this.props.filterAllowedCRS} additionalCRS={this.props.additionalCRS} changeInputValue={v => this.props.typeInput(v)}>
+                {list.map(crs =>
+                        <ListGroupItem
+                        key={crs.value}
+                        active={currentCRS === crs.value}
+                        onClick= { es => changeCrs(es.target.textContent)}
+                        eventKey={crs.value}
+                        >
+                            {crs.value}
+                        </ListGroupItem>)}
+        </CrsSelectorMenu>
+    </Dropdown> : null );
+    }
+}
+
+const crsSelector = connect(
+        createSelector(
+            currentBackgroundSelector,
+            projectionSelector,
+            projectionDefsSelector,
+            crsInputValueSelector,
+            modeSelector,
+            isCesium,
+            bottomPanelOpenSelector,
+                ( currentBackground, selected, projectionDefs, value, mode, cesium, bottomPanel) => ({
+                    currentBackground,
+                    selected,
+                    projectionDefs,
+                    value,
+                    enabled: (mode !== 'EDIT') && !cesium && !bottomPanel
+                })
+            ), {
+                typeInput: setInputValue,
+                setCrs: changeMapCrs,
+                onError: error
+            }
+        )(Selector);
+
+
+/**
+  * CRSSelector Plugin is a plugin that switches from to the pre-configured projections.
+  * it gets displayed into the mapFooter plugin
+  * @name CRSSelector
+  * @memberof plugins
+  * @class
+  * @prop {object[]} projectionDefs list of additional project definitions
+  * @prop {string[]} cfg.filterAllowedCRS list of allowed crs in the combobox list to used as filter for the one of retrieved proj4.defs()
+  * @prop {object} cfg.additionalCRS additional crs added to the list. The label param is used after in the combobox.
+  * @example
+  * // If you want to add some crs you need to provide a definition and adding it in the additionalCRS property
+  * // Put the following lines at the first level of the localconfig
+  * {
+  *   "projectionDefs": [{
+  *     "code": "EPSG:3003",
+  *     "def": "+proj=tmerc +lat_0=0 +lon_0=9 +k=0.9996 +x_0=1500000 +y_0=0 +ellps=intl+towgs84=-104.1,-49.1,-9.9,0.971,-2.917,0.714,-11.68 +units=m +no_defs",
+  *     "extent": [1241482.0019, 973563.1609, 1830078.9331, 5215189.0853],
+  *     "worldExtent": [6.6500, 8.8000, 12.0000, 47.0500]
+  *   }]
+  * }
+  * @example
+  * // And configure the new projection for the plugin as below:
+  * { "name": "CRSSelector",
+  *   "cfg": {
+  *     "additionalCRS": {
+  *       "EPSG:3003": { "label": "EPSG:3003" }
+  *     },
+  *     "filterAllowedCRS": ["EPSG:4326", "EPSG:3857"]
+  *   }
+  * }
+*/
+module.exports = {
+    CRSSelectorPlugin: assign(crsSelector, {
+        disablePluginIf: "{true}"
```

Comment:
disablePluginIf: "{true}" disables forever the plugin. We want only to remove it from default config (localConfig.js). Please remove this hard-coded disable